src/app/Main/main-dashboard/main-dashboard.component.ts

```diff
@@ -94,10 +93,10 @@ export class MainDashboardComponent implements OnInit {
   onupdateFunctionality(values: number) {
     this.updateShow = true;
   }
-  createOffer(){
-    this.service.createOffer(this.postForm.value).subscribe(res=>{
+  createOffer() {
+    this.service.createOffer(this.postForm.value).subscribe(res => {
```

Comment:
As a best practise we need to replace this deprecated subscription with the latest one.

.subscribe({
      next: (response) => {
        //code
      },
      error: (err: any) => {
       //code
      } 
    });